voidcrew/code/game/objects/structures/beach/serpent_tail.dm

```diff
@@ -0,0 +1,47 @@
+//Necropolis Tendrils, which spawn lavaland monsters and break into a chasm when killed
+/obj/structure/spawner/serpent
+	name = "serpent tail"
+	desc = "A tail of a serpent. Terrible monsters are pouring out from all around it."
+	icon = 'voidcrew/icons/mob/beach/serpent.dmi'
+	faction = list("beach")
+	max_mobs = 3
+	max_integrity = 650
+	mob_types = list(/mob/living/simple_animal/hostile/carp/megacarp)
+
+	move_resist = INFINITY
+	anchored = TRUE
+	resistance_flags = FIRE_PROOF | LAVA_PROOF
+
+	var/gps = null
+	var/obj/effect/light_emitter/tail/tail_emitted_light
```

Comment:
make sure to put a comment with `///` on the line before these and document what they do